streams/src/test/java/org/apache/kafka/streams/integration/ForeignKeyJoinSuite.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.streams.integration;
+
+import org.apache.kafka.common.utils.BytesTest;
+import org.apache.kafka.streams.kstream.internals.foreignkeyjoin.CombinedKeySchemaTest;
+import org.apache.kafka.streams.kstream.internals.foreignkeyjoin.SubscriptionResponseWrapperSerdeTest;
+import org.apache.kafka.streams.kstream.internals.foreignkeyjoin.SubscriptionWrapperSerdeTest;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+/**
+ * This suite runs all the tests related to the Suppression feature.
+ *
+ * It can be used from an IDE to selectively just run these tests when developing code related to Suppress.
+ *
+ * If desired, it can also be added to a Gradle build task, although this isn't strictly necessary, since all
+ * these tests are already included in the `:streams:test` task.
+ */
+@RunWith(Suite.class)
+@Suite.SuiteClasses({
+    BytesTest.class,
+    KTableKTableForeignKeyInnerJoinMultiIntegrationTest.class,
+    KTableKTableForeignKeyJoinIntegrationTest.class,
+    CombinedKeySchemaTest.class,
+    SubscriptionWrapperSerdeTest.class,
+    SubscriptionResponseWrapperSerdeTest.class
+})
+public class ForeignKeyJoinSuite {
```

Comment:
Added this for convenience; you can run this suite in an IDE to just blanket-test the whole "FK join" feature when making any changes.